client/src/components/Charts/SessionStats.jsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+import { Pie } from '@ant-design/plots';
+
+const URL = process.env.REACT_APP_URL;
+const USER_ID = 9999;
+
+const SessionStats = () => {
+  const [climberSessionStats, setClimberSessionStats] = useState();
+
+  const fetchClimberSessionStats = () => {
+    fetch(`${URL}/api/climbed_boulders_session/${USER_ID}/`)
+      .then(res => res.json())
+      .then(data => setClimberSessionStats(data));
+  };
+  useEffect(() => {
+    fetchClimberSessionStats();
+  }, []);
+
+  const config = {
+    appendPadding: 10,
+    data:
+      climberSessionStats?.filter(stat => stat.type != 'Total Climbs') || [],
```

Comment:
I think you could also work here with the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) instead of using [logical OR](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR). :)

And a question: Is the `!=` intended or is a `=`missing for `!==`?

Reply:
I'm afraid I don't know how to work with && in this case here. But I changed the != to !==. This was actually a mistake.